python/cudf/cudf/__init__.py

```diff
@@ -53,6 +53,7 @@
     UInt64Index,
     interval_range,
 )
+from cudf.core.default import no_default
```

Comment:
Why are we exposing this in the top level API of cudf, rather than mirroring pandas in `cudf.api.extensions.no_default`?

Reply:
Moved the `no_default` into `api.extensions` and removed it from top level API of `cudf` to closely mirror pandas now.